suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -477,7 +477,9 @@ export const getTxOperation = (
 };
 
 export const isNftTokenTransfer = (transfer: TokenTransfer) =>
-    ['ERC1155', 'ERC721'].includes(transfer.standard || '');
+    ['ERC1155', 'ERC721'].includes(transfer?.standard || '');
+
+export const isNftMultitokenTransfer = (transfer: TokenTransfer) => !!transfer?.multiTokenValues;
```

Comment:
are the `?` needed?

Reply:
nope

Reply:
but it is still there...

Reply:
weird, I was pretty sure I pushed it, here it is https://github.com/trezor/trezor-suite/pull/15457/commits/5a7283911c014c77fb8ebe47f29fe43f80752792